frontend/screens/Home.js

```diff
@@ -174,7 +151,35 @@ const styles = StyleSheet.create({
   buttonText: {
     textAlign: 'center',
     fontFamily: 'CircularStd-Bold',
-    fontSize: 18,
+    fontSize: normalize(18),
+  },
+  groupCreate: {
+    underlayColor: 'white',
+    backgroundColor: '#F15763',
+    borderRadius: 40,
+    width: width(0.5),
```

Comment:
is this right? syntax seems off for width

Reply:
also run make sure to run "eslint . --fix" after

Reply:
yeah it said width * 0.5 but was giving me an error when I ran that, so I changed it